plugins/package/cpb/src/main/kotlin/net/corda/cli/plugins/packaging/cpb/CreateCpbPluginWrapper.kt

```diff
@@ -0,0 +1,19 @@
+package net.corda.cli.plugins.packaging.cpb
+
+import net.corda.cli.api.CordaCliPlugin
+import org.pf4j.Extension
+import org.pf4j.Plugin
+import org.pf4j.PluginWrapper
+import picocli.CommandLine
+
+@Suppress("unused")
+class CreateCpbPluginWrapper(wrapper: PluginWrapper) : Plugin(wrapper) {
+
+    @Extension
+    @CommandLine.Command(
+        name = "create-cpb",
```

Comment:
As discussed offline with @josephzunigadaly create Cpb command should be made a sub command under existing packaging plugin.